auth0/resource_auth0_client.go

```diff
@@ -919,15 +947,103 @@ func flattenClientJwtConfiguration(jwt *management.ClientJWTConfiguration) []int
 }
 
 func flattenClientRefreshTokenConfiguration(refreshToken *management.ClientRefreshToken) []interface{} {
+	if refreshToken == nil {
+		return nil
+	}
+
 	m := make(map[string]interface{})
-	if refreshToken != nil {
-		m["rotation_type"] = refreshToken.RotationType
-		m["expiration_type"] = refreshToken.ExpirationType
-		m["leeway"] = refreshToken.Leeway
-		m["token_lifetime"] = refreshToken.TokenLifetime
-		m["infinite_token_lifetime"] = refreshToken.InfiniteTokenLifetime
-		m["infinite_idle_token_lifetime"] = refreshToken.InfiniteIdleTokenLifetime
-		m["idle_token_lifetime"] = refreshToken.IdleTokenLifetime
+
+	m["rotation_type"] = refreshToken.RotationType
+	m["expiration_type"] = refreshToken.ExpirationType
+	m["leeway"] = refreshToken.Leeway
+	m["token_lifetime"] = refreshToken.TokenLifetime
+	m["infinite_token_lifetime"] = refreshToken.InfiniteTokenLifetime
+	m["infinite_idle_token_lifetime"] = refreshToken.InfiniteIdleTokenLifetime
+	m["idle_token_lifetime"] = refreshToken.IdleTokenLifetime
+
+	return []interface{}{m}
+}
+
+func flattenClientAddons(addons map[string]interface{}) []interface{} {
```

Comment:
To properly fix the surfaced error we needed to add a custom flatten of the client addons.